src/Mod/Path/Gui/Resources/panels/PageOpPocketExtEdit.ui

```diff
@@ -107,7 +107,7 @@
          <item>
           <widget class="QPushButton" name="buttonDisable">
            <property name="toolTip">
-            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Diisable the currently selected pocket extension.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Disable the currently selected pocket extension.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Thanks for fixing this.

Reply:
No problem, mate. Was browsing and it caught my eye.

Reply:
Fixed